src/magento-payments/payments.md

```diff
@@ -1,3 +1,84 @@
 ---
 title: Magento Payments
----
\ No newline at end of file
+---
+
+Magento Payments is your turnkey self-service solution, including sandbox testing and a super simple setup, for providing robust and secure payment processing for your Magento websites.
+
+Whether you are a small business, mid-market contender, or large enterprise, our white-labeled pre-installed payments and risk solution will help you reduce operational overhead while increasing revenue, driving greater merchant adoption rates, and vastly improving the user's checkout experience.
+
+All the tools needed to implement and operate Magento Payments are fully integrated into the Magento Admin---manage everything in one spot with ease.
+
+Magento Payments is...
+
+* Easy to set up and maintain
+* Designed to maximize your profit
+* Safe and secure
+* Fully-featured to meet all your needs
+* Self-contained within the Magento Admin
+
+IMAGE
+
+## Features
+
+{:.bs-callout-info}
+Some of the features mentioned here may not yet be available for the Early Adopter Program (EAP).
+
+Magento Payments is your one-stop-shop for online checkout. We make setup and configuration painless and we provide powerful tools to give you the insight and control you need to create the best experience for your buyers.
+
+* [**Easy onboarding**]({% link magento-payments/onboarding-payments.md %})---Our process guides you through commercial signup, technical configuration, entitlements, sandbox environment configuration, and live payment enablement.
+* [**Payment checkout**]({% link magento-payments/configure-payments.md %})---Use pertinent local and region-specific payment methods to best serve your customers.
+* **Risk and fraud protection**---Leave the fraud prevention, chargebacks, and dispute management to us so that you can focus on your core business.
+* [**Cash flow management**]({% link magento-payments/financial-reporting.md %})---Synchronize payment details with orders to get full transparency to the processed volume, payment balance, and detailed transaction-level reporting for financial reconciliation.
+* **Analytics dashboard**---See the end-to-end customer journey across all payment options so you can glean the metrics you need to implement conversion improvements and optimize the customer experience.
+* **Transparent pricing**---Pricing is clear and upfront; what you see is what you get.
+
+<!-- ## Architectural overview
+
+See detailed information about the architecture of Magento Payments in the [Architecture topic in DevDocs](https://devdocs.magento.com/magento-payments/architecture.html). -->
+
+## Get started
+
+Onboarding and setting up Magento Payments is a quick process. Setup and configuration is completed in just a few steps:
+
+1. Connect your Magento instance to Magento Software as a Service (SaaS) services.
+1. Onboard and set up the sandbox service.
```

Comment:
(quibble) "set up the sandbox service" sounds a bit off for me - maybe something like "onboard to the Magento Payments sandbox service"
the next step could just be "Enable Magento Payment as your payment method and start processing test payments."
(end quibble)

Reply:
I totally understand this comment. My only thing is "onboard to the Magento Payments sandbox service" makes sense to us. But I'm not sure the average merchant understands that as well as they would when adding the "set up" part. I'll have to rethink this. Good feedback.